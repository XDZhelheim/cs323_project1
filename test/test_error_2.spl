int test_error_2() {
    int 2a = 0xFFFZFFFF;
    b = 1
    if (a > b) {
        c[1 = 2;
    }
}