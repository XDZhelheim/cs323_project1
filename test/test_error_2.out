Error type A at Line 2: unknown lexeme 2a
Error type A at Line 2: unknown lexeme 0xFFFZFFFF
Error type B at Line 3: Missing semicolon ';'
Error type B at Line 5: Missing closing parenthesis ']'
